lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -538,3 +543,182 @@ def test_move_users_to_same_cohort(self):
             ],
             verify_order=False
         )
+
+
+@patch('instructor_task.tasks_helper.DefaultStorage', new=MockDefaultStorage)
+class TestGradeReportCertificateInfo(TestReportMixin, InstructorTaskModuleTestCase):
+    """
+    Test that grade report has correct user certificate information.
```

Comment:
"Test that grade report has correct user enrolment, verification, and certificate information."